MapleServer2/PacketHandlers/Game/StateSkillHandler.cs

```diff
@@ -3,25 +3,29 @@
 using MapleServer2.Servers.Game;
 using Microsoft.Extensions.Logging;
 
-namespace MapleServer2.PacketHandlers.Game {
-    public class StateSkillHandler : GamePacketHandler {
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StateSkillHandler : GamePacketHandler
+    {
         public override ushort OpCode => RecvOp.STATE_SKILL;
 
         public StateSkillHandler(ILogger<StateSkillHandler> logger) : base(logger) { }
 
-        public override void Handle(GameSession session, PacketReader packet) {
+        public override void Handle(GameSession session, PacketReader packet)
+        {
             byte function = packet.ReadByte();
-            if (function == 0) {
+            if (function == 0)
+            {
                 // This count seems to increase for each skill used
-                int counter = packet.ReadInt();
+                packet.ReadInt();   // Counter
```

Comment:
Why remove the variable declaration just to add the variable name as a comment next to it?

Reply:
Removing variable that is not being used, but add the comment to keep track for what it was using it in further implementations.